tests/prescribed_temperature_in_field.prm

```diff
@@ -0,0 +1,64 @@
+## First we set up a unit length box model with uniform everything.
+set Dimension   = 2
+set End time    = 10000
+# Load the signal library.
+set Additional shared libraries = /Users/danieldouglas/software/aspect/cookbooks/prescribed_temperature/./libprescribed_temperature.so
```

Comment:
```suggestion
set Additional shared libraries = tests/libprescribed_temperature.so
```

Reply:
I need to start handling my files better, I thought I had already done this but it must have been on a different branch!